source/includes/_tipser-widget.md

```diff
@@ -155,6 +207,8 @@ lang | `'en'` | a locale to be used by the Tipser content. Possible values: `'en
 env | `'prod'` | a Tipser environment to be used by the Tipser content. Possible values: `'stage'` and `'prod'`. | `'stage'`
 disableDomReplacement | `false` | Advanced setting. Set to true in case for some reason you don't wish any tag replacement to happen (see: [Replacing elements on your page](#replacing-elements-on-your-page) ). | true
 defaultAddedToCartPopup | `true` | Controls default Added To Cart Popup. It appears when user adds a product to the cart. It improves UX by highlighting the action and allowing to navigate quickly to the cart modal window.  | `true` or `false` 
+useDefaultErrorHandler | `true` | set it to false if you dont want to get error popup with message | see [Adding event handlers](#adding-event-handlers)
```

Comment:
description should be the same as for `Tipser-element IMO

Reply:
ok